lxd/container_lxc.go

```diff
@@ -7126,6 +7128,14 @@ func (c *containerLXC) IsPrivileged() bool {
 	return shared.IsTrue(c.expandedConfig["security.privileged"])
 }
 
+func (c *containerLXC) IsDevLxd() bool {
+	if (c.expandedConfig["security.devlxd"] == "") {
+		return true
+	} else {
+		return shared.IsTrue(c.expandedConfig["security.devlxd"])
+	}
```

Comment:
The checks look wrong. It should probably be:

```
enableDevLxd, ok := c.expandedConfig["security.devlxd"]
if !ok {
        return false
}

if enableDevLxd == "" {
        return false
}

return shared.IsTrue(enableDevLxd)
```

Reply:
I was returning true in the case that c.expandedConfig["security.devlxd"] == "" so that the /dev/lxd configuration would still be in the container by default.  If I understand correctly, we only want to disable /dev/lxd if the "-c security.devlxd=false" is set when creating a container.  Is this correct?

Reply:
Oh, is that what you discussed with @stgraber, that it would be enabled by default? If this is security sensitive I'd expect this to be `false` by default but maybe this is a backwards compatibility concern of his that I missed.

Reply:
Right, @stgraber apparently wants this on by default so that part is good then.

Reply:
Yes that is what I discussed with @stgraber .  He said "Yeah, it (security.devlxd) should be true by default so that we keep the current behavior by default."

Reply:
Ok thank you.